src/web/layouts/CommentLayout.tsx

```diff
@@ -438,7 +438,7 @@ export const CommentLayout = ({
                     </GridItem>
                     <GridItem area="right-column">
                         <RightColumn>
-                            <StickyAd />
+                            <StickyAd name="right" height={1059} />
```

Comment:
Just curious, where does the value 1059 comes from ? :)

Reply:
+1 named var here would be useful context